api/src/utils/defaultValidate.js

```diff
@@ -0,0 +1,9 @@
+const Joi = require("joi");
+
+function validateId(id) {
+  return Joi.string().allow(null, "").validate(id, { stripUnknown: true });
+}
```

Comment:
Not sure we should allow null, WDYT?

```suggestion
function validateId(id) {
  return Joi.string().validate(id, { stripUnknown: true });
}
```

Reply:
Je suis sûr j'ai d'ailleurs eu un bug à cause de ça j'avais oublié de te demander !

Reply:
Attends du coup j'ai pas compris ta réponse, on doit autoriser null et vide sinon ça bug ?